style.css

```diff
@@ -501,14 +493,56 @@ footer {
 }
 
 @media screen and (max-width: 768px) {
- .nav-links {
+  .nav-links {
     display: flex;
-    flex-direction: row;
-    gap: 12px;
+    flex-direction: column;
+    gap: 50px;
+    border-bottom: 1px solid #fff;
+    justify-content: center;
+    align-items: flex-start;
+    height: 100vh;
```

Comment:
- [ ] [REQUIRED] Please, make sure to always stick to the Figma Design 🖼️ as requested in the project requirement 🧾. I can see that your design is working perfectly 👍 but please change the colors to match the Figma Design ♻️.

Your Design | Figma Design
------------|--------------
![image](https://user-images.githubusercontent.com/60979926/169644754-1456f710-b5c2-4898-a43b-01dda39e0d61.png) | ![image](https://user-images.githubusercontent.com/60979926/169644859-8f08fec7-80a9-40fa-a6d2-81580fe29dfa.png)